spec.html

```diff
@@ -36049,6 +36071,116 @@ <h1>
+          </h1>
+          <dl class="header">
+          </dl>
+          <emu-alg>
+            1. If _S_ is the empty String, return 0.
+            1. Let _codepoints_ be StringToCodePoints(_S_).
+            1. Let _eUTF_ be the smallest index into _S_ that corresponds to the character at element _e_ of _codepoints_. If _e_ is greater than or equal to the number of elements in _codepoints_, then _eUTF_ is the number of code units in _S_.
+            1. Return _eUTF_.
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-match-records">
+          <h1>Match Records</h1>
+          <p>A <dfn variants="Match Records">Match Record</dfn> is a Record value used to encapsulate the start and end indices of a regular expression match or capture.</p>
+          <p>Match Records have the fields listed in <emu-xref href="#table-match-record"></emu-xref>.</p>
+          <emu-table id="table-match-record" caption="Match Record Fields">
+            <table>
+              <tr>
+                <th>Field Name</th>
+                <th>Value</th>
+                <th>Meaning</th>
+              </tr>
+              <tr>
+                <td>[[StartIndex]]</td>
+                <td>a non-negative integer</td>
+                <td>The number of code units from the start of a string at which the match begins (inclusive).</td>
+              </tr>
+              <tr>
+                <td>[[EndIndex]]</td>
+                <td>an integer &ge; [[StartIndex]]</td>
+                <td>The number of code units from the start of a string at which the match ends (exclusive).</td>
+              </tr>
+            </table>
+          </emu-table>
+        </emu-clause>
+
+        <emu-clause id="sec-getmatchstring" type="abstract operation">
+          <h1>
+            GetMatchString (
+              _S_: a String,
+              _match_: a Match Record,
+            )
+          </h1>
+          <dl class="header">
+          </dl>
+          <emu-alg>
+            1. Assert: _match_.[[StartIndex]] is an integer value &ge; 0 and &le; the length of _S_.
+            1. Assert: _match_.[[EndIndex]] is an integer value &ge; _match_.[[StartIndex]] and &le; the length of _S_.
+            1. Return the substring of _S_ from _match_.[[StartIndex]] to _match_.[[EndIndex]].
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-getmatchindexpair" type="abstract operation">
+          <h1>
+            GetMatchIndexPair (
+              _S_: a String,
+              _match_: a Match Record,
+            )
+          </h1>
+          <dl class="header">
+          </dl>
+          <emu-alg>
+            1. Assert: _match_.[[StartIndex]] is an integer value &ge; 0 and &lt; the length of _S_.
+            1. Assert: _match_.[[EndIndex]] is an integer value &ge; _match_.[[StartIndex]] and &le; the length of _S_.
+            1. Return ! CreateArrayFromList(&laquo; 𝔽(_match_.[[StartIndex]]), 𝔽(_match_.[[EndIndex]]) &raquo;).
```

Comment:
This operation no longer fails. You should rebase and correct any errors like this that ecmarkup would now catch.

```suggestion
            1. Return CreateArrayFromList(&laquo; 𝔽(_match_.[[StartIndex]]), 𝔽(_match_.[[EndIndex]]) &raquo;).
```